20240923.md

```diff
@@ -0,0 +1,68 @@
+# Meeting Minutes
+
+**Cardano Atlantic Council Meeting**
+
+**Date:** Septemer 23, 2024 - 18:30 UTC
+
+## **Attendees:** 
+- Jenny Brito
+- Adam Dean
+- Beatrice Anihiri
+- Richard McCracken
+- Thiago Nunes
+- Mike Hornan
+
+## **Purpose:** 
+- To dicuss and possibly reach a decision on the current info action known as the Ikigai proposal.
+- Prerequisite to the meeting was the reading and evaluation of the proposal prior to the meeting.
+- Setting up a legal entity.
+
+## **Discussion Points:**
+
+1. **Process to help decision making**
+- How to prevent group think and confirmation bias in the decision making process.
+- Preliminary voting thoughts on the current active Ikigai proposal.
+- Discussion on order of operations, tentative votes and people's thoughts.
+- First round the room for individucals voting preferences and rationales.
+- Review of the proposal and the relevant sections of the Interim Constitution.
+
+2. **Round 2 of discussion and voting**
+- Secend round the room temperature check to see if we have consensus.
+- Further discussion of the pros and cons of a Yes, No or Abstain vote.
+- Further review of the proposal and relevant secions of the Interim Constitution.
+- Possible Abstain decsion.
+
+
+3. **Disseneting Rationale**
+- The dissent to a Yes vote as opposed to an abstain vote provides support for the content of the proposal. This was determined to be a double edge sword since the purpose of the ICC is determine the 
```

Comment:
```suggestion
- The dissent to a Constitutional vote as opposed to an Abstain vote provides support for the content of the proposal. This was determined to be a double edge sword since the purpose of the ICC is determine the 
```

Reply:
Also, seems like this point is a bit... unfinished?